addons/website_slides/models/slide_channel.py

```diff
@@ -216,6 +233,39 @@ def message_post(self, parent_id=False, subtype=None, **kwargs):
                 subtype = 'mail.mt_note'
         return super(Channel, self).message_post(parent_id=parent_id, subtype=subtype, **kwargs)
 
+    # ---------------------------------------------------------
+    # Rating Mixin API
+    # ---------------------------------------------------------
+
+    def action_add_member(self, target_user=False, **member_values):
```

Comment:
Would be better if it was "target_partner" as discussed.

Reply:
Zizisse done.